src/scripts/main.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
-}
+  return new Promise(resolve => {
+    element.addEventListener(`${eventName}`, () => {
+      resolve(`
+      It was ${eventName} on the element:${element.nodeName}, id: ${element.id}
```

Comment:
```suggestion
      It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}.
```
Fix it to pass the tests. You can split it in two string if line is too long

Reply:
> Thanks, now it passes the test !